backend/lib/services/firestore_service.dart

```diff
@@ -2,20 +2,41 @@ import 'package:backend/utils/type_utils.dart';
 import 'package:googleapis/firestore/v1.dart';
 import 'package:googleapis_auth/auth_io.dart';
 
+import '../extensions/json_list_extension.dart';
+
+/// console: https://console.firebase.google.com/project/news-curation-project/firestore/data
+// we will hit this endpoint every (say) hour, with a cron job: https://cloud.google.com/scheduler
 class FirestoreService {
   FirestoreService(AutoRefreshingAuthClient client, {FirestoreApi? api}) {
-    _api = api ?? FirestoreApi(client);
+    var backendDefine = const String.fromEnvironment('BACKEND');
```

Comment:
I believe this can be a bit cleaner
```suggestion
    const backendDefine = String.fromEnvironment('BACKEND');
```

Reply:
Nice one!  Thanks mate!